plugins/cue_points/annotation/lib/api/KalturaAnnotation.php

```diff
@@ -56,6 +56,21 @@ class KalturaAnnotation extends KalturaCuePoint
 	 * @readonly
 	 */
 	public $directChildrenCount;
+	
+	/**
+	 * Is the annotation public.
+	 * @var KalturaNullableBoolean
+	 * @filter eq
+	 */
+	public $isPublic;
+	
+	/**
+	 * Should the cue point get indexed on the entry.
+	 * @var KalturaNullableBoolean
+	 */
+	public $shouldIndexOnEntry;
```

Comment:
Consider to rename to searchScopes and make it comma separated enum values (CUE_POINT, ENTRY). We might have additional scopes in the future.